### DISCUSSION TITLE
An error occurred while running task08.fp(vasp)

### DISCUSSION THREAD

**Discussion by kunda2022** [deepmodeling/dpgen#793] [Q&A]
Is the INCAR wrong or something happened? 
The OUTCAR context as below:                
 running on   64 total cores
 distrk:  each k-point on   64 cores,    1 groups
 distr:  one band on    1 cores,   64 groups
 using from now: INCAR     
 vasp.6.1.0 28Jan20 (build Mar 23 2020 15:34:42) complex                        
  
 POSCAR found type information on POSCAR Be
 POSCAR found :  1 types and      48 ions
                                                                         
|     Error reading item LREAL from file INCAR.                               |
|      Error code was IERR= 5 ... . Found N= 3 data.                          |

And err as following:
INFO:dpgen:Try to parse from vasprun.xml
INFO:dpgen:Failed fp path: iter.000000/02.fp/task.000.000048/
INFO:dpgen:Try to parse from vasprun.xml
INFO:dpgen:Failed fp path: iter.000000/02.fp/task.000.000049/
INFO:dpgen:failed frame:     50 in     50  100.00 % 
raise RuntimeError("find too many unsuccessfully terminated jobs. Too many FP tasks are not converged. Please check your input parameters (e.g. INCAR) or configuration (e.g. POSCAR) in directories \'iter.*.*/02.fp/task.*.*/.\'")
RuntimeError: find too many unsuccessfully terminated jobs. Too many FP tasks are not converged. Please check your input parameters (e.g. INCAR) or configuration (e.g. POSCAR) in directories 'iter.*.*/02.fp/task.*.*/.'

[dpgen.log](https://github.com/deepmodeling/dpgen/files/9081333/dpgen.log)
[err.txt](https://github.com/deepmodeling/dpgen/files/9081334/err.txt)
[INCAR.txt](https://github.com/deepmodeling/dpgen/files/9081335/INCAR.txt)
[machine.json.txt](https://github.com/deepmodeling/dpgen/files/9081337/machine.json.txt)
[OUTCAR.txt](https://github.com/deepmodeling/dpgen/files/9081338/OUTCAR.txt)
[param.json.txt](https://github.com/deepmodeling/dpgen/files/9081339/param.json.txt)
[record.dpgen.txt](https://github.com/deepmodeling/dpgen/files/9081340/record.dpgen.txt)
[POSCAR.txt](https://github.com/deepmodeling/dpgen/files/9081383/POSCAR.txt)

**Comment by kunda2022** [comment]
When I changed the LREAL = auto, a new error in OUTCAR was  presented. 
Error reading item KGAMMA from file INCAR.   
Error code was IERR= 5 ... . Found N= 0 data.

**Comment by HuangJiameng** [comment]
Please check your POTCAR files.
